.github/workflows/k8s-ci.yml

```diff
@@ -3,11 +3,11 @@ name: K8s-CI
 on:
   # Skipping pre-merge K8s tests for now
   # https://github.com/elastic/cloudbeat/issues/1623
-  # pull_request:
-  #   branches:
-  #     - main
-  #     - "[0-9]+.[0-9]+"
-  #   types: [ opened, synchronize, reopened ]
+  pull_request:
+    branches:
+      - main
+      - "[0-9]+.[0-9]+"
+    types: [ opened, synchronize, reopened ]
```

Comment:
@oren-zohar, after enabling `k8s-ci` on pre-merge, it appears to be stable and functioning well. Are we planning to continue with `pre-merge` tests, or is there consideration to enable the full version of tests only on post-merge? Additionally, we've rerun flaky tests workflow, which has contributed to improved stability.